src/material/config.bzl

```diff
@@ -15,12 +15,14 @@ entryPoints = [
     "dialog",
     "dialog/testing",
     "divider",
+    "divider/testing",
```

Comment:
It's consistent, but it feels a little silly to have a test harness for what is basically a `div`.

Reply:
Yeah its a little silly, but there are actually couple methods on it, and just having the selector encapsulated in a harness is kind of nice